sanic/response.py

```diff
@@ -3,7 +3,8 @@
 from os import path
 from urllib.parse import quote_plus
 
-from aiofiles import open as open_async  # type: ignore
+#from aiofiles import open as open_async  # type: ignore
+from trio import open_file as open_async
```

Comment:
The easy way'd be defining those incompatible imports in `compat.py` module, then import things from compat.

Reply:
Somewhat better now but detection of environment by argv is far from optimal...